executor/adapter.go

```diff
@@ -267,15 +268,15 @@ func (a *ExecStmt) buildExecutor(ctx context.Context) (Executor, error) {
 func (a *ExecStmt) logSlowQuery() {
 	cfg := config.GetGlobalConfig()
 	costTime := time.Since(a.startTime)
-	sql := a.Text
+	sql := []rune(a.Text)
```

Comment:
I just feel converting the whole text to rune just to avoid the last split character is not efficient.

Reply:
So, any better suggestion?

Reply:
```
for {
    if utf8.RuneStart(str[length]) {
	    break
	}
    length--
}
```

Reply:
any problems with this approach?

sql = fmt.Sprintf("%.*s(len:%d)", cfg.Log.QueryLogMaxLen, sql, len(sql))

Reply:
@Darren 
Good solution, this format syntax I've never seen before, and we can change `*s` to `*q`.

Reply:
`%.*s` or `%.*q` can't work as expected @coocood @Darren